deploy/charts/trust-manager/values.yaml

```diff
@@ -352,6 +352,9 @@ podDisruptionBudget:
 # Labels to apply to all resources
 commonLabels: {}
 
+# Annotations to apply to all resources
```

Comment:
Maybe a small note on the fact that annotations are not currently added to CRDs?